tor-button@shatur/files/tor-button@shatur/applet.js

```diff
@@ -94,10 +105,10 @@ MyApplet.prototype = {
     
     on_applet_clicked: function() {
         if (torAppletCheck.substr(5,3) == "OFF") { // Check information from file without 'true,' and '\n '
-                                                   GLib.spawn_command_line_async(this.location); // Run tor from location settings        
+                                                   GLib.spawn_command_line_async(gksu+this.location); // Run tor from location settings        
         }
         else {
-            GLib.spawn_command_line_async('pkill tor');  // Kill Tor                 
+            GLib.spawn_command_line_async(gksu+'pkill tor');  // Kill Tor                 
```

Comment:
I think there's two issues here. ```pkill``` is not really a default app people have installed, and I think the source of errors like this:

```
pkill: killing pid 404 failed: Operation not permitted
pkill: killing pid 421 failed: Operation not permitted
```

is caused by pkill killing the wrong process, that also happen to be owned by root.

Also, left-clicking the applet icon (which again, I would not expect to immediately _kill_ Tor), just hides the Tor window and displays this when I try to start a new process:

![screenshot from 2017-08-06 14-18-28](https://user-images.githubusercontent.com/6859057/29006296-956b6998-7ab2-11e7-8aff-a28352db0df2.png)

Right after you kill Tor, this displays:

![screenshot from 2017-08-06 14-22-23](https://user-images.githubusercontent.com/6859057/29006301-bfa69688-7ab2-11e7-94b8-6bfb31036363.png)

I think you either should either not kill Tor, or use the Muffin API to do it.